File: Assignments/2/A3_NUTHANNAIK_CS19B051/ques2/tworay_isotropic_2.py
import math


def function():
	pt = 50.00
	ht = 50
	hr = 2
	v = []
	for i in range(100, 1001, 200):
		pr = (pt *((ht*hr)**2))/(i**4)
		pr = 30 + 10*(math.log10(pr))
		v.append(pr)
	freq2 = 100
	for i in range(len(v)):
		print(str(freq2) + " " + str(v[i]))
		freq2 += 200
	print()

if __name__ == '__main__':
	function()

File: Assignments/2/A3_AKHIL_CS19B053/3.2/tworay_isotropic_2.cpp
#include<bits/stdc++.h>
using namespace std;
int main(){
double Pt=50.00;
double Ht=50;
double Hr=2;
vector<double>v;
for(double dist=100;dist<=1000;dist+=200){
    double Pr;
    Pr=(double)(Pt)*(pow(Ht*Hr,2))/pow(dist,4);
    Pr=30+10*log10(Pr);
    v.push_back(Pr);
}
int freq2=100;
int k=0;
for(int i=0;i<v.size();i++){
    cout<<freq2<<" "<<v[i]<<endl;
    freq2+=200;
}
cout<<endl;
}
